docs/next/guides/building-and-testing/modules.md

```diff
@@ -264,32 +57,32 @@ import {
   PasswordEditor,
   SelectEditor,
   TextEditor,
+  registerEditor, // editors' registering method
 } from 'handsontable/editors';
 
 // import renderers
 import {
-  registerRenderer,
   baseRenderer,
   autocompleteRenderer,
   checkboxRenderer,
   htmlRenderer,
   numericRenderer,
   passwordRenderer,
   textRenderer,
+  registerRenderer, // renderers' registering method
```

Comment:
In this whole document, `registering method` should be replaced with `registering function`. Because these are functions, not methods.

There might be more mistakenly used instances of the word "method" throughout the docs.

See: https://flaviocopes.com/javascript-difference-method-function/

Reply:
Ok, done: 8f61684